src/Uno.UI.RemoteControl.Host/HotReload/ServerHotReloadProcessor.cs

```diff
@@ -60,7 +63,7 @@ private async Task ProcessConfigureServer(ConfigureServer configureServer)
 				.Select(p => new FileSystemWatcher
 				{
 					Path = p,
-					Filter = "*.xaml",
+					Filter = "*.*",
```

Comment:
Is it because temporary files are of a diffrent suffix?

Reply:
Yes, and the operation comes in as a rename and the filter seems to be on the source name, not the target name.